console/src/main/java/org/matrix/console/activity/ImageSliderActivity.java

```diff
@@ -77,6 +82,21 @@ public void transformPage(View view, float position) {
         }
     }
 
+    /**
+     * Return the used MXSession from an intent.
+     * @param intent
+     * @return the MXsession if it exists.
+     */
+    private MXSession getSession(Intent intent) {
+        String matrixId = null;
+
+        if (intent.hasExtra(EXTRA_MATRIX_ID)) {
+            matrixId = intent.getStringExtra(EXTRA_MATRIX_ID);
+        }
```

Comment:
You needn't do the `hasExtra` check here. `intent.getStringExtra` will return `null` if it wasn't set as per http://developer.android.com/reference/android/content/Intent.html#getStringExtra(java.lang.String)